### PR TITLE
More words and such

### PR DESCRIPTION
**Pull request by chrisjrn**
Anyone want to do a pass over the copy changes I've made?

### CODE REVIEW

File: pinaxcon/templates/static_pages/about/north_bay_python.html [hunk 1]
```diff
@@ -9,29 +9,47 @@
...
-
-A single-track conference north of the Bay focused on community, collaboration, and all things Python.
-
+  A single-track conference north of the Golden Gate, focused on community, collaboration, and all things Python.
 {% endblock %}
 
 {% block content %}
 
-<h2>Welcome to the North Bay</h2>
+<p>North Bay Python is a two-day, single-track Python conference held in the Mystic Theatre in Historic Downtown Petaluma, California, held over the weekend of December 2 &amp; 3, 2017.</p>
```
Comment (bluesomewhere) [inline pinaxcon/templates/static_pages/about/north_bay_python.html]: "held in the" -> "held at the"

File: pinaxcon/templates/static_pages/about/north_bay_python.html [hunk 2]
```diff
@@ -9,29 +9,47 @@
...
 
-<p>North Bay Python is a two-day, single-track Python conference held in the Mystic Theater in Petaluma, California, held over the weekend of December 2 &amp; 3, 2017.</p>
+<p>We're a nonprofit conference for professionals, enthusiasts and students alike; we're focused on inclusion, accessibility, diversity, and affordability; and we're planning a great lineup of talks from all over the Python ecosystem, with plenty of time to meet new people and
```
Comment (bluesomewhere) [inline pinaxcon/templates/static_pages/about/north_bay_python.html]: gotta break this sentence up

File: pinaxcon/templates/static_pages/about/north_bay_python.html [hunk 3]
```diff
@@ -9,29 +9,47 @@
...
+  develop new ideas too.</p>
 
-<p>We're a nonprofit conference for professionals, enthusiasts and students alike, focused on inclusion, accessibility, diversity, and affordability, with not only a great lineup of talks from all over the Python ecosystem but plenty of time to meet new people and develop new ideas too.</p>
-
-<p>Our venue, the historic Mystic Theater in Downtown Petaluma, is not only a beautiful example of an early 1900s Vaudeville theatre, but is a short walk away from over fifty different food and drink options in Downtown Petaluma, as well as several different hotels, the closest of which is only a block away.</p>
+<p>Our venue, the historic Mystic Theatre in Downtown Petaluma, is a beautiful example of an early 1900s Vaudeville theatre, and it's a short walk away from over fifty different food and drink options in Downtown Petaluma, as well as several different hotels, the closest of which is only a block away.</p>
```
Comment (bluesomewhere) [inline pinaxcon/templates/static_pages/about/north_bay_python.html]: "fifty" -> 50, and mebbe drop historic as well as "in Downtown Petaluma" as it seems redundant

File: pinaxcon/templates/static_pages/about/north_bay_python.html [hunk 4]
```diff
@@ -9,29 +9,47 @@
...
 
 <h2>The Conference</h2>
 
-<p>North Bay Python is a single-track conference, with a carefully curated set of talks representing the truly diverse Python community and their different areas of interest. If a topic is less to your interest, or you've met some people you really want to sit down and chat with, we'll have plenty of areas away from the main theater to catch up and chat as well.</p>
+<p>North Bay Python is a single-track conference, with a carefully curated set of talks representing the truly diverse Python community and their different areas of interest.
```
Comment (bluesomewhere) [inline pinaxcon/templates/static_pages/about/north_bay_python.html]: missing </p>? comma not necessary, also drop "truly"

File: pinaxcon/templates/static_pages/about/north_bay_python.html [hunk 5]
```diff
@@ -9,29 +9,47 @@
...
+<p>
+
+<p>If a topic is less to your interest, or you've met some people you really want to sit down and chat with, we'll have plenty of areas away from the main theatre to catch up and chat as well.</p>
```
Comment (bluesomewhere) [inline pinaxcon/templates/static_pages/about/north_bay_python.html]: drop "as well"

File: pinaxcon/templates/static_pages/about/north_bay_python.html [hunk 6]
```diff
@@ -9,29 +9,47 @@
...
+
+<h2>Petaluma, California</h2>
+
+<p>North Bay Python's home is Petaluma, a delightfully quaint dairy town, nestled on a river at the southern edge of California's Wine Country. We've got beautiful scenery right on our doorstep, and we're less than an hour's drive from San Francisco over the Golden Gate Bridge.</p>
+
+<p>The Mystic is not the only local example of early 1900s architecture, either; downtown is full of historic buildings, and it's at the center of the maker movement and a thriving craft brewery scene.</p>
```
Comment (bluesomewhere) [inline pinaxcon/templates/static_pages/about/north_bay_python.html]: split into two separate sentences

File: pinaxcon/templates/static_pages/about/north_bay_python.html [hunk 7]
```diff
@@ -9,29 +9,47 @@
...
+
+<h3>Getting here</h3>
```
Comment (bluesomewhere) [inline pinaxcon/templates/static_pages/about/north_bay_python.html]: title case

File: pinaxcon/templates/static_pages/about/north_bay_python.html [hunk 8]
```diff
@@ -9,29 +9,47 @@
...
+
+<h4>By car</h4>
+
+<p>If you're driving up, Downtown Petaluma is at exit 472 on Highway 101, 35 miles north of the Golden Gate Bridge. All parking is free in Petaluma, including in the undercover garages at Keller St and Theatre Square, both are in short walking distance of the Mystic.</p>
```
Comment (bluesomewhere) [inline pinaxcon/templates/static_pages/about/north_bay_python.html]: Last clause should be a sentence of its own. "Both garages are in..."

File: pinaxcon/templates/static_pages/about/north_bay_python.html [hunk 9]
```diff
@@ -9,29 +9,47 @@
...
 
-<p>Petaluma is right in the middle of the beautiful scenery of the North Bay, and less than an hour's drive from San Francisco over the Golden Gate Bridge. The Mystic is not the only example of early 1900s architecture, either; downtown is full of historic buildings, as well as being at the center of thing things like the maker movement and a thriving craft brewery scene.</p>
+<h3>Staying here</h3>
 
-<p>If you want to stay overnight, we have deals with local hotels - the closest one is just one block away. If you're driving up, Petaluma has plenty of free parking, and if you don't feel like driving, we'll be putting on a shuttle from public transport stops as well.</p>
+<p>Petaluma, as well as being a place where people live, also has hotels! We're arranging deals with some of the best local hotels in the area &ndash; the closest is just one block away. We'll share details of our hotel deals with you when conference tickets go on sale.</p>
```
Comment (bluesomewhere) [inline pinaxcon/templates/static_pages/about/north_bay_python.html]: drop the "as well" aside, it's nice but unwieldy. "share details of our hotel deals" -> "share details"

**Comment by chrisjrn** [comment]
I've addressed all of the changes requested by @joshsimmons.

**Review by andrewgodwin — APPROVED** [review]
Looks good to me from a first read through.

**Review by bluesomewhere — APPROVED** [review]

**Pull request merged**